.github/workflows/CI.yaml

```diff
@@ -147,15 +166,48 @@ jobs:
 
       - name: "Test: Gear workspace"
         run: ./scripts/gear.sh test gear --release
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: JS metadata"
         run: ./scripts/gear.sh test js
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+
+      - name: "Test: Process gear-test"
+        run: ./scripts/gear.sh test gtest
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: Process node runtime testsuite"
         run: ./scripts/gear.sh test rtest
 
       - name: "Test: Runtime upgrade and queue processing"
         run: ./scripts/gear.sh test runtime-upgrade
+            
+      - name: "Coverage: Aggregate"
```

Comment:
There is an issue appeared for other PR, where coverage app fails job on small diff coverage, need to fix it

Reply:
It will be fixed when codecov.yml will be in master.

Reply:
> It will be fixed when codecov.yml will be in master.

Great. Can't wait to merge it 😇